notebooks/02-ingestion/02-batch-ingestion.ipynb

```diff
@@ -631,6 +631,95 @@
     "\"\"\")"
    ]
   },
+  {
```

Comment:
A note that this notebook has the SQL keywords in `CODE` though we've been asked to keep them capitalized but note in `pre` any more. Maybe a good time to update that if you have the time :D

Reply:
done